src/components/MyKiva/BlogCard.vue

```diff
@@ -68,18 +69,7 @@ const props = defineProps({
 
 const optimizedImageUrl = computed(() => {
 	if (!props.image) return '';
-
-	// blog card dimensions: w:336px, h:55% = ~200px
-	const width = 336;
-	const height = 200;
-
-	// check if it's a Contentful URL
-	const isContentfulUrl = props.image.includes('images.ctfassets.net')
-		|| props.image.includes('assets.contentful.com');
-	if (!isContentfulUrl) {
-		return props.image;
-	}
-	return `${props.image}?w=${width}&h=${height}&fm=webp&q=80`;
+	return optimizeContentfulUrl(props.image, 336, 200);
```

Comment:
Same comment about height.

Reply:
@dyersituations Pushing up the unit test code as we speak- also yes, testing locally soon so will remove or keep height values. I'm pushing my code just to ensure my edits are being documented